Sources/Misc/RawDataContainer.swift

```diff
@@ -11,14 +11,33 @@
 //
 //  Created by Nacho Soto on 11/16/21.
 
-/// A type which exposes its underlying raw data, for debugging purposes or for getting access
+/// A type which exposes its underlying content for debugging purposes or for getting access
 /// to future data while using an older version of the SDK.
 public protocol RawDataContainer {
 
-    /// The type of the underlying raw data
-    associatedtype Content
+    /// The type of the `underlyingData` for this type.
```

Comment:
I think you get automatic linking to the symbol if you do \`\`RawDataContainer/underlyingData\`\` here